p2/p2.py

```diff
@@ -0,0 +1,52 @@
+class Problem2:
+    def __init__(self,arr):
+        self.arr = arr
+
+    def element_at(self, index):
+        try:
+            if index >= len(self.arr):
+                return "index out of bounds"
```

Comment:
can create a method to validate inputs instead of re-writing the code, follow DRY principle